src/main/java/uk/nhs/ctp/service/dto/TriageQuestion.java

```diff
@@ -37,25 +36,14 @@
 	private String enableWhenQuestionnaireId;
 	private boolean enableWhenAnswer;
 
-	public List<TriageOption> getOptions() {
```

Comment:
Does it matter that the default value of getOptions will be null rather than empty list? Should this be initialised in the case where no options have been added?

Reply:
I'll add an initialiser on the model, but from what I can tell we never try to read these apart from when theyre displayed on the ui.